blocks/library/freeform/freeform-block.js

```diff
@@ -405,6 +405,7 @@ export default class FreeformBlock extends Component {
 			</BlockControls>,
 			<TinyMCE
 				key="editor"
+				className={ className }
```

Comment:
I guess I'm not understanding why a block must always have the default classname.